src/nxdoc/nuxeo-server/indexing-and-query/configuring-the-elasticsearch-mapping.md

```diff
@@ -243,75 +285,39 @@ To do case insensitive search using an `ILIKE` operation you need to declare you
 
 ## Making STARTSWITH Work with a Custom Field
 
-To use a `STARTSWITH` operator on a field with a path pattern like a hierarchical vocabulary. Turn your field into a `multi_field` and a `children` sub field:
+To use a `STARTSWITH` operator on a field with a path pattern like a hierarchical vocabulary. Turn your field into a multi field with a `children` sub field:
 
 ```
 "my:field" : {
-  "type" : "multi_field",
+  "type" : "keyword",
   "fields" : {
-    "my:field" : {
-      "index" : "not_analyzed",
-      "type" : "string"
-    },
     "children" : {
+      "type" : "text",
       "search_analyzer" : "keyword",
-      "index_analyzer" : "path_analyzer",
-      "type" : "string"
+      "index_analyzer" : "path_analyzer"
```

Comment:
In our config we have `analyzer` rather than `index_analyzer`, does it matter?

Reply:
yes I had to change this in the mapping so here also thx